src/modules/controllers/list.controller.js

```diff
@@ -52,6 +52,5 @@ module.exports.del = (req, res) => {
       res.send(err);
```

Comment:
`Task.find().then((result2) => {`
Where is Task from? And result2 - incorrect name